src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/OverloadResolutionResult.cs

```diff
@@ -973,7 +973,7 @@ private static bool HadLambdaConversionError(DiagnosticBag diagnostics, Analyzed
                         sourceLocation,
                         symbols,
                         arg + 1,
-                        refParm.ToArgumentDisplayString());
+                        refParm.ToParameterDisplayString());
```

Comment:
Not yet. There are many areas in binding that we're still adding tests to.
I'll add a test that covers this instance for now.